evaluator/builtin_string.go

```diff
@@ -463,6 +463,23 @@ func builtinTrim(args []types.Datum, _ context.Context) (d types.Datum, err erro
 	return d, nil
 }
 
+// For LTRIM & RTRIM
+// See: https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_ltrim
+// See: https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_rtrim
+func trimFn(fn func(string, string) string, cutset string) BuiltinFunc {
```

Comment:
```trimFn``` here is just a high-order function. It's used to build trimLeft and trimRight. Maybe we could add following lines to make it more clear:
``` go
var (
  builtinLTrim = trimFn(strings.TrimLeft, spaceChars)
  builtinRTrim = trimFn(strings.TrimRight, spaceChars)
)
```